odl/solvers/functional/default_functionals.py

```diff
@@ -2193,6 +2198,197 @@ def __repr__(self):
                                          self.__norm.pwisenorm.exponent)
 
 
+class IndicatorSimplex(Functional):
```

Comment:
Add the functional to `__all__`, to make it available under `odl.solvers` directly.